src/toil/batchSystems/abstractBatchSystem.py

```diff
@@ -47,6 +48,12 @@ def __init__(self, config, maxCores, maxMemory, maxDisk):
         """
         :type dict[str,str]
         """
+        self.workerCleanupInfo = {
```

Comment:
Easy isn't always better. I prefer a namedtuple because it makes two things obvious: 1) the immutability and 2) the fact that the set of keys is static across al instances. It also opens the door for adding behavior via methods. The fix is easy, just move the definition to the global scope. If you are worried about proximity between definition and use, I recommend moving the clean-up stuff towards the end of the class and have the namedtuple definition immediately below the class.